[TRACING-3556] Document disable_keep_alives (#70668)
openshift/openshift-docs: modules/otel-config-collector.adoc

        tls: # <2>
        headers: # <3>
          X-Scope-OrgID: "dev"
        disable_keep_alives: false <4>

    service:
      pipelines:
        traces:
          exporters: [otlphttp]
        metrics:
          exporters: [otlphttp]
----
<1> The OTLP HTTP endpoint. If the `+https://+` scheme is used, then client transport security is enabled and overrides the `insecure` setting in the `tls`.
<2> The client side TLS configuration. Defines paths to TLS certificates.
<3> Headers are sent in every HTTP request.

> <4> If true, disables HTTP keep-alives. It will only use the connection to the server for a single HTTP request.

Reviewer: ```suggestion
<4> If true, disables HTTP keep-alive mode. The parameter only uses the connection to the server for a single HTTP request.
```

* On scanning the customer documentation, "keep-alive" is usually followed by another noun that better describes the component. I also suggest removing "it" as it's a vague referrent. I assume we are discussing the parameter "uses the connection..."?
